CMakeLists.txt

```
@@ -115,7 +115,12 @@ if (VULKAN_HEADERS_ENABLE_INSTALL)
     install(DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/registry" DESTINATION "${CMAKE_INSTALL_DATADIR}/vulkan" USE_SOURCE_PERMISSIONS)
 
     set_target_properties(Vulkan-Headers PROPERTIES EXPORT_NAME "Headers")
-    install(TARGETS Vulkan-Headers EXPORT VulkanHeadersConfig INCLUDES DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+    if (VULKAN_HEADERS_ENABLE_MODULE)
+        set_target_properties(Vulkan-HppModule PROPERTIES EXPORT_NAME "HppModule")
+        install(TARGETS Vulkan-Headers Vulkan-HppModule EXPORT VulkanHeadersConfig INCLUDES DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+    else()
+        install(TARGETS Vulkan-Headers EXPORT VulkanHeadersConfig INCLUDES DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+    endif()
```

Logic might be simpler to read if there was a list of install targets

```suggestion
    list(APPEND INSTALL_TARGETS Vulkan-Headers)
    if (VULKAN_HEADERS_ENABLE_MODULE)
            set_target_properties(Vulkan-HppModule PROPERTIES EXPORT_NAME "HppModule")
            list(APPEND INSTALL_TARGETS Vulkan-HppModule)
    endif()
    install(TARGETS ${INSTALL_TARGETS} EXPORT VulkanHeadersConfig INCLUDES DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
```

---

That looks a lot better, you're right!